scale/gf_logs.py

```diff
@@ -0,0 +1,18 @@
+"""This module is handeling the logging of plus one scale in gluster."""
+
+
+import logging
+
+# logger decides what to log
+glogger = logging.getLogger(__name__)
+glogger.setLevel(logging.INFO)
+
+# file handler decides where and how to log those logs of logger
+gf_log_handler = logging.FileHandler("/var/log/glusterfs/gluster-plus-one-scale.logs")
+
+format_string = '%(asctime)s:%(funcName)s:%(levelname)s:%(lineno)d:%(message)s'
```

Comment:
Use same format as used in other gluster components, so that it will be easy to parse using the external tools. `<TIME> <LEVEL> [<LINENUM>:<FUNCNAME>] <MESSAGE>`